deepmd/pt/entrypoints/main.py

```diff
@@ -377,6 +387,111 @@ def show(FLAGS):
             log.info(f"The fitting_net parameter is {fitting_net}")
 
 
+def change_bias(FLAGS):
```

Comment:
I suggest not passing `FLAGS` but expanding the dict.

Reply:
Other functions in this file have similar issues but can be handled in a separated PR.

Reply:
Tracked in https://github.com/deepmodeling/deepmd-kit/issues/3934.